static/app/views/releases/detail/overview/issues.tsx

```diff
@@ -108,20 +161,25 @@ class Issues extends Component<Props, State> {
     switch (issuesType) {
       case IssuesType.ALL:
         return {
-          path: `/organizations/${orgId}/issues/`,
+          path: `/organizations/${organization.slug}/issues/`,
           queryParams: {
             ...queryParams,
             query: stringifyQueryObject(new QueryResults([`release:${version}`])),
           },
         };
       case IssuesType.RESOLVED:
         return {
-          path: `/organizations/${orgId}/releases/${version}/resolved/`,
-          queryParams: {...queryParams, query: ''},
+          path: `/organizations/${organization.slug}/issues/`,
+          queryParams: {
+            ...queryParams,
+            query: stringifyQueryObject(
+              new QueryResults([`release:${version}`, 'is:resolved'])
+            ),
+          },
```

Comment:
This is not the same. I think your change will show you all issues that are now resolved that are happening in this release. The previous code is actually what we want - the issues resolved in(by) this release.